### ISSUE TITLE
Bug: Adding Feeds - submit button doesn't propagate onclick event in Firefox

### ISSUE THREAD

**Issue by maciej-lech** [MISP/MISP#9002]
### Expected behavior

When adding a feed via `<misp host>/feeds/add` form, after click `Submit`, the feed should be added (POST request) and the browser redirects to the `Feeds List` page.

### Actual behavior

After click Submit, nothing happen. No logs, no requests, nothing in the browser's developer console.

### Steps to reproduce

Try to add a feed via GUI using the given versions.

What is surprising, after unsuccessful submit, if I click `Filter rules: Modify` and then `Cancel` in the popup window, the previous `onclick` event is propagated, and the feed is submitted as it supposed to be in a first place.

Another workaround is to click `Inspect (Q)` on the `Submit` button and then right-click on that button tag in the `Inspector` windows and then click `Use in Console`. Then, if I run `.onclick()` method manually, everything seems to work fine.

### Version

2.4.167

### Operating System

Ubuntu

### Operating System version

22.04

### PHP version

7.3

### Browser

Firefox

### Browser version

112.0+build1-0ubuntu0.22.04.1~mt1

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maciej-lech** [comment]
Editing an already added feed doesn't seem to have this problem. The submit button works fine there.

**Comment by Nicolas-Pellletier** [comment]
Yes i confirm it thanks to share it. It wasn't easy for me to see the workaround. Hope it will help others that have struggled like me
